app/src/components/misc/ExtensionInstalled/index.tsx

```diff
@@ -2,14 +2,21 @@ import React, { useEffect } from "react";
 import { useSelector } from "react-redux";
 import { Navigate } from "react-router-dom";
 import { trackExtensionInstalled } from "modules/analytics/events/misc/installation";
-import { getExtensionInsallSource } from "store/selectors";
+import { getAppMode, getExtensionInsallSource } from "store/selectors";
+import { initImplicitTestRuleWidgetConfig } from "components/features/rules/TestThisRule";
 import PATHS from "config/constants/sub/paths";
 import { useFeatureValue } from "@growthbook/growthbook-react";
 
 const ExtensionInstalled: React.FC = () => {
   const installSourcePage = useSelector(getExtensionInsallSource);
   const onboardingVariation = useFeatureValue("activation_without_onboarding", "variant");
 
+  const appMode = useSelector(getAppMode);
+
+  useEffect(() => {
+    initImplicitTestRuleWidgetConfig(appMode);
```

Comment:
What about initializing the config when user logs out? Do we sync this config?
consider the case:
1. use logs out
2. his storage gets cleared
3. he logs back in -> this config wont be present and not initialize as well